resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -126,6 +126,8 @@ package object config extends Logging {
       .stringConf
       .createOptional
 
+  private[spark] val KUBERNETES_DRIVER_ENV_KEY = "spark.driverEnv."
```

Comment:
I don't see how this can be handled in a generic way across all cluster managers. Where is the code that handles this in spark-core? Otherwise we should set the environment variables specifically on the executor container in KubernetesClusterSchedulerBackend.